run_test.sh

```diff
@@ -57,6 +63,18 @@ setup_and_build_happ() {
   hc-scaffold link-type certificate:EntryHash like --delete false --bidirectional false
   hc-scaffold link-type agent:creator post:EntryHash --delete false --bidirectional true
 
+  if [[ -n "$OVERRIDE_HOLOCHAIN_VERSION" ]]; then
+    nix develop --override-input "versions/holochain" github:holochain/holochain/$OVERRIDE_HOLOCHAIN_VERSION --command bash -c "
+      set -e
```

Comment:
small non-blocking suggestion: could this sequence of commands and the identical one just below it be DRYed up into a separate function? Same goes for line 98 onwards.